scheds/rust/scx_rusty/src/bpf/main.bpf.c

```diff
@@ -561,6 +564,41 @@ static u64 sched_prio_to_latency_weight(u64 prio)
 	return sched_prio_to_weight[DL_MAX_LAT_PRIO - prio - 1];
 }
 
+static void do_iowait_boost(struct task_struct *p, struct task_ctx *taskc, s32 cpu)
+{
+	if (!iowait_boost || !p || !taskc || cpu < 0)
```

Comment:
So I don't _think_ we should need to check `!p || !taskc` here. This is a static subprog, so BPF should be able to track that these pointers are trusted based on the context of the caller. If the verifier complains without it, I'd be curious to know why.

For `cpu < 0` -- I don't think we need to validate that here either, though not for verifier reasons. If the value is `< 0`, then we've made some mistake in the caller and it's probably OK / desired to have the scheduler get booted out when we call `__COMPAT_scx_bpf_cpuperf_set()` with an invalid CPU.